probe_scraper/scrapers/moz_central_scraper.py

```diff
@@ -272,7 +276,7 @@ def scrape(folder=None, min_fx_version=None, max_fx_version=None, channels=None)
         folder = tempfile.mkdtemp()
 
     error_cache = load_error_cache(folder)
-    requests_cache.install_cache('probe_scraper_cache')
+    requests_cache.install_cache("probe_scraper_cache")
     results = defaultdict(dict)
 
     if channels is None:
```

Comment:
Looks to me like this is still scraping all the channels. We should explicitly only scrape release.

Reply:
My mistake, keep as-is